src/components/Menu/HomeMenu.tsx

```diff
@@ -338,7 +338,7 @@ class HomeMenu extends React.Component<Props, State> {
                               (a) => a.location === this.props.location.pathname
                             );
                           return (
-                            <div key={item.location} className="linkContainer">
+                            <li key={item.location} className="linkContainer">
```

Comment:
Nit: for semantic use of reactstrap, I think this should be a `<NavItem/>`

https://reactstrap.github.io/components/navbar/

Reply:
done